api/source/service/mysql/STIGService.js

```diff
@@ -358,12 +369,6 @@ exports.queryRules = async function ( ruleId, inProjection ) {
       where rgr.ruleId = r.ruleId) as "fixes"`)
   }  
 
-  if ( inProjection && inProjection.includes('detail') ) {
-    columns.push(
-      'r.iacontrols'
-    )
-  }
-
 
   // CONSTRUCT MAIN QUERY
   let sql = 'SELECT '
```

Comment:
Please include the groupBy array joined ",\n" here

Reply:
removed .toJSON() and added groupBy